portfolio-form-submission.js

```diff
@@ -20,8 +21,8 @@ const setDateLimits = () => {
 };
 const submitFunc = (event) => {
   event.preventDefault();
-  
```

Comment:
instead of useinf getElementbyId many times you can make common function like

getElement(id){
document.getElementById(id)
}

let responseMsg =  getElement(resp-message)

and the same for req.query

this improves code quality